Cargo.toml

```diff
@@ -56,6 +57,7 @@ rustls-webpki-roots = ["tokio-rustls", "ring", "webpki", "webpki-roots"]
 rustls-native-roots = ["tokio-rustls", "ring", "rustls-native-certs"]
 
 [dev-dependencies]
+futures-util = { version = "0.3.14", default-features = false, features = ["sink"] }
```

Comment:
We don't enforce minvers for dev-dependencies so this could be abbreviated to `0.3` but I don't mind either way